truss/templates/base.Dockerfile.jinja

```diff
@@ -18,7 +18,8 @@ RUN $PYTHON_EXECUTABLE -c "import sys; \
 
 {% block install_uv %}
 {# Install `uv` if not already present in the image. #}
-RUN if ! command -v uv >/dev/null 2>&1; then (curl -LsSf https://astral.sh/uv/{{ UV_VERSION }}/install.sh | sh) >/dev/null 2>&1; fi
+RUN apt update && apt install -y curl
```

Comment:
We do install `curl` later as [well](https://github.com/basetenlabs/truss/blob/3c0f5597ed5cca223f21e2bb337514d750ef51a4/truss/templates/server.Dockerfile.jinja?plain=1#L21), but I think it's ok to duplicate just the portion we need here